### ISSUE TITLE
Prestige timer reseting

### ISSUE THREAD

**Issue by Amoulier** [Tune389/Tap-Titans-2-Progress-Bot#94]
Has the title said, sometimes the prestige timer got reset to "0" without stopping the script, making the prestige time double or triple on some times.

**Comment by chrisreyn** [comment]
Hiro crashes because of "lack of memory" issue as it states in its log, so it restarts. Changing default Nox launcher to a different one helps **improve** the emulator, but it may still lead to Hiro crashing eventually. i think it's just a Nox problem hoping Nox dev works on it, but idk how to reach them

**Comment by atereshkov** [comment]
I have never seen this bug before.

**Comment by snapclick** [comment]
yeah I got this problem also and I was thinking that something was wrong with my clock lol

**Comment by Amoulier** [comment]
Did what @chrisreyn suggested and the prestige timer keep resetting by himself.

**Comment by walterhuangio** [comment]
@Amoulier you can check hiromacro for logs, and then you'll see if the macro app got restarted. Do you have 4 core + 4GB for your emulator (It's high I know, Nox is not optimized for 5.1 yet)?

**Comment by Amoulier** [comment]
@walterhuangio  here is my last log of my emulator...  the system where I'm running the emulator have 24 cores and 64 gb ram,  so i have the emulator running with 4 cores and 5 gb ram.

Has you can see, the script has restarted more than two times!

[log.txt](https://github.com/Tune389/Tap-Titans-2-Progress-Bot/files/1910829/log.txt)

**Comment by chrisreyn** [comment]
increasing ram for Nox does NOT help in anyway. something inherent to Nox is broken with handling memory

**Comment by Amoulier** [comment]
alright the problem has been solve.  Just installed the latest MEmu, set it up, and everything is running perfectly.

**Comment by walterhuangio** [comment]
@Amoulier go MEmu! Install NOVA launcher your life will be even better.

**Comment by sheng0821** [comment]
MEmu 480x800 dpi how much?
